Drag.py

```diff
@@ -0,0 +1,189 @@
+also used as a value that is satisfyingly close to zero. Hard-coded (can be changed).
+'''
+delta = .00001
+# Max velocity cannot be initialized to 0 or there will be an initial divide-by-zero error
+Max_Velocity = .1
+Bouyancy = 0
+# Magnitude of applied force for determining drag coeffcients in linear axes
+appliedLinearForce = rospy.get_param("LINEAR_FORCE", default=10)
+# Magnitude of applied torque for determining drag coeffcients in rotational axes
+appliedTorque = rospy.get_param("TORQUE", default=5)
+appliedForceDown = rospy.get_param('ForceDown', default=-20)
+TimeOfAplliedForceDown = rospy.get_param('TimeOfForceDown')
+
+
+def Find_Bouyancy(choice):  # Initial function used to upward force of buoyancy (used in determining drag in Z axis)
+    global Bouyancy
+    Down_Force = -.5  # Applies initial downward force in z axis
+    pub = rospy.Publisher('/wrench', WrenchStamped, queue_size=30)
+    rospy.init_node('move_sub', anonymous=False)
+    force_msg = WrenchStamped()
+    force_msg.wrench.force.z = appliedForceDown
+    pub.publish(force_msg)  # publish wrench with force
+    rospy.sleep(TimeOfAplliedForceDown)  # node sleeps for some amount of time before continuing
+    force_msg.wrench.force.z = 0  # Applies 0 force which stops downward force
+    pub.publish(force_msg)
+    while not (rospy.is_shutdown()):
+        rospy.Subscriber("/odom", Odometry, get_velocity, choice)
+        rospy.sleep(1)
+        if Velocity > 0.0:
+            while not (rospy.is_shutdown()):
+                force_msg.wrench.force.z = Down_Force
+                pub.publish(force_msg)
+                Down_Force = Down_Force - .001
+                rospy.sleep(.01)
+                if Velocity < 0.0:
+                    break
+            break
+    Bouyancy = abs(Down_Force)
+
+
+def get_velocity(data, choice):  # Function sets velocity in a certain axis depending on char input
+    global Velocity
+    # linear
+    if choice == 'x':
+        Velocity = data.twist.twist.linear.x
+    elif choice == 'y':
+        Velocity = data.twist.twist.linear.y
+    elif choice == 'z':
+        Velocity = data.twist.twist.linear.z
+    # rotational
+    elif choice == 'rl':
+        Velocity = data.twist.twist.angular.x
+    elif choice == 'p':
+        Velocity = data.twist.twist.angular.y
+    elif choice == 'yw':
+        Velocity = data.twist.twist.angular.z
+
+
+def Calculate_Drag(choice):
+    '''
+    Calculates drag based on the initial applied force and the approximate max velocity
+    the sub achieves in that axis. See for formula:
+    Axis determined by char argument.
+    '''
+    global Linear_Drag_X, Linear_Drag_Y, Linear_Drag_Z, Pitch_Drag, Roll_Drag, Yaw_Drag, Max_Velocity, Bouyancy
+    if (choice == 'x'):
+        Linear_Drag_X = (appliedLinearForce / abs(Max_Velocity))
+    elif (choice == 'y'):
+        Linear_Drag_Y = (appliedLinearForce / abs(Max_Velocity))
```

Comment:
I don't see a `from __future__ import division` at the top of the file. This can lead to some annoying bugs in the future if an int is passed into this function.  If you still want to keep the int dividing behavior in other parts of the file, cast one of the values to a `float` before dividing.